packages/components/src/resizable-box/index.tsx

```diff
@@ -112,6 +112,12 @@ function UnforwardedResizableBox(
 				showHandle && 'has-show-handle',
 				className
 			) }
+			handleComponent={ Object.fromEntries(
+				Object.keys( HANDLE_CLASSES ).map( ( key ) => [
+					key,
+					<div key={ key } tabIndex={ -1 } />,
+				] )
+			) }
```

Comment:
How is this related to the PR?

Reply:
I wouldn't do this if it was't. 😃 Unfortunately `re-resizable` doesn't create proper focusable element for the handles, so the focus goes to the block wrapper instead when you drag them, which triggers block drag.

Reply:
I'm wondering about `tabIndex={ -1 }`. This seems like it will remove the handles from the tab sequence, which if I'm not wrong is something we want to keep (for example in the handles of the site editor frame resizer and maybe others)

Reply:
> This seems like it will remove the handles from the tab sequence

There is no tab sequence, these handles are currently completely unfocusable, that's my point :)

Reply:
The site editor one is within the tab sequence (in trunk) but I confirmed that it's still within the tab sequence after this change.

Reply:
Ah, yes, it seems to have its own `handleComponent` override